searvey/ioc.py

```diff
@@ -162,6 +212,18 @@ def normalize_ioc_stations(df: pd.DataFrame) -> gpd.GeoDataFrame:
     )
     return gdf
 
+def normalize_ioc_stations_api(df: pd.DataFrame) -> gpd.GeoDataFrame:
+    df = df.assign(
+        # fmt: off
+        GlossID=df.GlossID.astype(pd.Int64Dtype()),
+        observations_ratio_per_month=df.observations,
```

Comment:
I think that with this line we will end up with two identical columns: `observations` and `observations_ratio_per_month`.

If we want to only change the name, then `df = df.rename({...})` should be prefered.

 `df = df.assign(...)` is used for creating new columns and/or applying some transformation on a existing column. E.g.
 ``` python
 df = df.assign(gloss_id=df.gloss_id.astype(int)) # we are changing the data type and replacing the existing column
 ```